mmdet/models/dense_heads/sabl_retina_head.py

```diff
@@ -600,11 +600,11 @@ def get_bboxes_single(self,
                 bbox_cls_pred.contiguous(),
                 bbox_reg_pred.contiguous()
             ]
-            bboxes, confids = self.bbox_coder.decode(
+            bboxes, confidences = self.bbox_coder.decode(
                 anchors.contiguous(), bbox_preds, max_shape=img_shape)
             mlvl_bboxes.append(bboxes)
             mlvl_scores.append(scores)
-            mlvl_confids.append(confids)
+            mlvl_confids.append(confidences)
```

Comment:
do we also need to change `mlvl_confid` -> `mlvl_confidences`>

Reply:
yes